src/web/app/components/teammates-common/collapse-anim.ts

```diff
@@ -1,17 +1,15 @@
-import { animate, AnimationTriggerMetadata, state, style, transition, trigger } from '@angular/animations';
+import { animate, AnimationTriggerMetadata, style, transition, trigger } from '@angular/animations';
 
 /**
  * Directive for collapsing of *ngIf columns.
  */
 export const collapseAnim: AnimationTriggerMetadata = trigger('collapseAnim', [
-  state('*', style({ overflow: 'hidden' })),
-  state('void', style({ overflow: 'hidden' })),
   transition(':leave', [
-    style({ height: '*' }),
+    style({ height: '*', overflow: 'hidden' }),
     animate('300ms ease-in-out', style({ height: 0, opacity: 0 })),
   ]),
   transition(':enter', [
-    style({ height: '0' }),
-    animate('300ms ease-in-out', style({ height: '*', opacity: 1 })),
+    style({ height: '0', overflow: 'hidden' }),
+    animate('300ms ease-in-out', style({ height: '*', opacity: 1, overflow: 'visible' })),
```

Comment:
Dont think `overflow: 'visible'` does anything, since `style` within `animate` gets removed after the transition.
If the aim is to be explicit, you could add `state('*', style({ overflow: 'visible' }))` instead.
Refer:  https://stackoverflow.com/a/47351870

Reply:
Thanks for the link! I think it is not necessary to be explicit so I have removed it.